solidity/integration/smoke_test.js

```diff
@@ -1,12 +1,7 @@
-const BondedECDSAKeepVendor = artifacts.require("./BondedECDSAKeepVendor.sol")
-const BondedECDSAKeepVendorImplV1 = artifacts.require(
-  "./BondedECDSAKeepVendorImplV1.sol"
-)
 const BondedECDSAKeepFactory = artifacts.require("./BondedECDSAKeepFactory.sol")
 const BondedECDSAKeep = artifacts.require("./BondedECDSAKeep.sol")
 
 const RandomBeaconService = artifacts.require("IRandomBeacon")
-const {RandomBeaconAddress} = require("../migrations/external-contracts")
 
```

Comment:
Are we sure we want to modify this file? It looks like the linter went crazy. I don't think it's going to work with the changes here.

Reply:
Hmm they were marked as unused and I didn't see any usage in fact. But OK, I will restore them.